atlas/AtlasMl/atlasml/ml/clustering.py

```
@@ -0,0 +1,70 @@
+from enum import Enum
+import numpy as np
+from typing import Tuple, Optional
```

Ruff (0.12.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)